storages/storages.go

```diff
@@ -61,10 +60,10 @@ type Storages struct {
 
 // Backend represents any storage in akubra cluster
 type Backend struct {
-	http.RoundTripper
-	Endpoint    url.URL
-	Name        string
-	Maintenance bool
+	RoundTripper transport.Matcher
```

Comment:
http.RoundTripper

Reply:
transport.Matcher type in this case is required for successful compilation.
We no need type casting in this implementation.